src/core/public/chrome/chrome_service.tsx

```diff
@@ -165,12 +167,41 @@ export class ChromeService {
     // Can delete
     const getNavType$ = uiSettings.get$('pageNavigation').pipe(takeUntil(this.stop$));
 
+    const isIE = () => {
+      const ua = window.navigator.userAgent;
+      const msie = ua.indexOf('MSIE '); // IE 10 or older
+      const trident = ua.indexOf('Trident/'); // IE 11
+
+      return msie > 0 || trident > 0;
+    };
+
     if (!this.params.browserSupportsCsp && injectedMetadata.getCspConfig().warnLegacyBrowsers) {
-      notifications.toasts.addWarning(
-        i18n.translate('core.chrome.legacyBrowserWarning', {
-          defaultMessage: 'Your browser does not meet the security requirements for Kibana.',
-        })
-      );
+      notifications.toasts.addWarning({
+        title: mountReactNode(
+          <FormattedMessage
+            id="core.chrome.legacyBrowserWarning"
+            defaultMessage="Your browser does not meet the security requirements for Kibana."
+          />
+        ),
+      });
```

Comment:
Any reason we're using FormattedMessage here instead of the `i18n.translate` API like before?

Reply:
Since the other one needed to be switched I figured it better to convert this one over to the same format so that we didn't need the extra import?

Reply:
Ah for the link, that's right. Makes sense!